io/buffer.ts

```diff
@@ -168,15 +167,14 @@ export class Buffer {
     this.#off = 0;
     this.#reslice(Math.min(m + n, MAX_SIZE));
     return m;
-  };
-
-  /** Grows the buffer's capacity, if necessary, to guarantee space for
+  } /** Grows the buffer's capacity, if necessary, to guarantee space for
```

Comment:
This seems fishy.

Reply:
`deno fmt` mangled it, I know how to reproduce it so will open an issue.